glossary.md

```diff
@@ -12,17 +12,17 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | English                           | Tiếng Việt (Nếu bạn bổ sung thì KHÔNG căn chỉnh các dòng khác) |
 |-----------------------------------|----------------------------------------------------------------|
 | accuracy                          | độ chính xác                                                   |
-| benchmark                         |                                                                |
+| benchmark                         | chuẩn so sánh                                                  |
```

Comment:
benchmark không hẳn là chuẩn so sánh. Nó giống như là có bảng so sánh các/nhiều thuật toán với nhau với cùng điều kiện, chỉ số đánh giá

Reply:
A vẫn vote là **đánh giá xếp hạng**.

Reply:
How about *bảng xếp hạng* (cúp C1)

Reply:
Mình chuyển thành "đánh giá xếp hạng" theo @sonvx giống như bên [này](https://docs.google.com/spreadsheets/d/1Hcd4AqR7Xzd08Ws0gwhZ-LX6XNzJKgCDy0VcFc3D66c/edit#gid=0) luôn